language/tools/move-cli/src/sandbox/commands/publish.rs

```diff
@@ -1,16 +1,20 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::sandbox::utils::{
-    explain_publish_changeset, explain_publish_error, get_gas_status,
-    on_disk_state_view::OnDiskStateView,
+use crate::{
+    sandbox::utils::{
+        explain_publish_changeset, explain_publish_error, get_gas_status,
+        on_disk_state_view::OnDiskStateView,
+    },
+    NativeFunctionRecord,
 };
 use move_lang::{self, compiled_unit::CompiledUnit, Compiler, Flags};
 use move_vm_runtime::move_vm::MoveVM;
 
 use anyhow::Result;
 
 pub fn publish(
+    natives: Vec<NativeFunctionRecord>,
```

Comment:
Nit: consider `IntoIterator` or a slice (since `MoveVM` doesn't need ownership?)

Reply:
Well the VM does need ownership here. This record contains fully qualified function names that will be used as keys in collections. With that being said, I'll switch it to `impl IntoIterator<NativeFunctionRecord>`, just to make it more general.